project/src/components/logo/logo.tsx

```diff
@@ -0,0 +1,11 @@
+import {Link} from 'react-router-dom';
+
+function Logo(): JSX.Element {
+  return (
+    <Link className='header__logo-link header__logo-link--active' to='/'>
+      <img className='header__logo' src={'img/logo.svg'} alt='6 cities logo' width='81' height='41'/>
```

Comment:
у тебя здесь просто строка)
```suggestion
      <img className='header__logo' src='img/logo.svg' alt='6 cities logo' width='81' height='41'/>
```